lib/events.js

```diff
@@ -75,7 +75,7 @@ EventEmitter.init = function() {
 // Obviously not all Emitters should be limited to 10. This function allows
 // that to be increased. Set to zero for unlimited.
 EventEmitter.prototype.setMaxListeners = function setMaxListeners(n) {
-  if (typeof n !== 'number' || n < 0 || isNaN(n)) {
+  if (typeof n !== 'number' || n < 0 || Number.isNaN(n)) {
```

Comment:
I think the `typeof n !== 'number'` check is redundant if `Number.isNaN` is being used.